packages/flutter/lib/src/widgets/overscroll_indicator.dart

```diff
@@ -26,7 +26,8 @@ import 'ticker_provider.dart';
 /// [GlowingOverscrollIndicator] generates [OverscrollIndicatorNotification]
 /// before showing an overscroll indication. To prevent the indicator from
 /// showing the indication, call [OverscrollIndicatorNotification.disallowGlow]
-/// on the notification.
+/// on the notification. In order to paint the glow at a different offset,
+/// modify [OverscrollIndicatorNotification.paintOffset] of the notification.
 ///
 /// Created automatically by [ScrollBehavior.buildViewportChrome] on platforms
 /// (e.g., Android) that commonly use this type of overscroll indication.
```

Comment:
Further below (starting at line 42ish) can we change this to something like:

```dart
/// To manipulate the position of the GlowingOverscrollIndicator in this case, 
/// you can provide a [OverscrollIndicatorNotification.paintOffset] to the 
/// notification, or use a [NestedScrollView].
```

Reply:
Done.